src/components/ConversationSettings/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -70,7 +70,11 @@
 						:container="container"
 						@close="closeLogModal">
 						<div class="modal__content">
-							<textarea v-model="processLog" class="log-content" />
+							<NcTextArea :value="processLog"
+								class="log-content"
+								:label="t('spreed', 'Log content')"
+								rows="29"
+								resize="vertical" />
```

Comment:
Shouldn't it be either `:value.sync` as a replacement for `v-model`, or a `readonly` textarea if it is not supposed to be editable?

Reply:
Yeah, it was writable but if it is just log, then it makes sense to keep it read only.